pennylane_qulacs/qulacs_device.py

```diff
@@ -26,11 +41,16 @@
 Z = np.array([[1, 0], [0, -1]])
 H = np.array([[1, 1], [1, -1]])/np.sqrt(2)
 SWAP = np.array([[1, 0, 0, 0], [0, 0, 1, 0], [0, 1, 0, 0], [0, 0, 0, 1]])
-CSWAP = block_diag(I, I, SWAP)
 
 rx = lambda theta: np.cos(theta / 2) * I + 1j * np.sin(-theta / 2) * X
 ry = lambda theta: np.cos(theta / 2) * I + 1j * np.sin(-theta / 2) * Y
 rz = lambda theta: np.cos(theta / 2) * I + 1j * np.sin(-theta / 2) * Z
```

Comment:
These can likely be deleted now, since `Rot` has been removed from the operation dictionary. Perhaps also the constant matrices above (I, X, Y, Z, H, SWAP)?